examples/dreambooth/train_dreambooth.py

```diff
@@ -289,6 +294,14 @@ def parse_args(input_args=None):
     parser.add_argument(
         "--enable_xformers_memory_efficient_attention", action="store_true", help="Whether or not to use xformers."
     )
+    parser.add_argument("--samples_per_checkpoint", type=int, default=0, help="Whether or not to save samples for every checkpoint specified by --checkpointing_steps.")
+    parser.add_argument("--sample_steps", type=int, default=40, help="Number of steps for generating sample images.")
+    parser.add_argument("--sample_prompt", type=str, default=None, help="Prompt to use for sample image generation.")
+    parser.add_argument("--sample_seed", type=int, default=-1, help="Seed for the per-checkpoint sample image generation. -1 to select random seed")
+    parser.add_argument("--convert_checkpoints", action="store_true", help="Auto-convert checkpoints to an inference ready structure")
```

Comment:
Let's also not add this. We will handle this soon directly in accelerate as well, see: https://github.com/huggingface/diffusers/pull/2048